recipe/activate.bat

```diff
@@ -129,7 +129,7 @@ CALL "VC\Auxiliary\Build\vcvars%BITS%.bat" -vcvars_ver=@{vcvars_ver} %WindowsSDK
 :: if this didn't work and CONDA_BUILD is not set, we're outside
 :: conda-forge CI so retry without vcvars_ver, which is going to
 :: fail on local installs that don't match our exact versions
-if %ERRORLEVEL% neq 0 exit 1 (
+if %ERRORLEVEL% neq 0 (
```

Comment:
Wow, I must have seriously messed up when rebasing out the other changes here. Thanks. Let's see if this changes something, but I believe I had it correct initially and it still failed...